packages/rancher-logging/rancher-logging.patch

```diff
@@ -74,4 +74,6 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-logging/charts-original/values
 +  address: ""
 +  cluster: true
 +  root_ca: ""
-\ No newline at end of file
++
++additionalLoggers:
++  RKE1Enabled: false
```

Comment:
Since we are creating an additional Logging component with this flag enabled, I think we should name it with `Logging` rather than `Logger`. Also, it may be possible that more fields other than `enabled` are exposed here and a comment would be helpful to explain what this field does.

Here's my suggestion for reformatting it:
```yaml
## Configure which default Logging CRs are deployed to capture logs from other locations
##
additionalLoggingSources:
    rke:
        enabled: true
```